### DISCUSSION TITLE
Informix - change database thru SQL not reflect on dbeaver selection

### DISCUSSION THREAD

**Discussion by ceinmart** [dbeaver/dbeaver#35827] [Q&A] [answered]
### Description

That works before. Stopped to work few versions ago.
If changes the database selection using a SQL command like `database xpto` the dbeaver interface doesn't update .

I have this set on my driver settings , which I don't remember anymore if I set it manually or if it became as default. 
Anyway, as I mentioned before, this resource has worked in the past.

![image](https://github.com/user-attachments/assets/c56e8439-8ca2-4ddc-b105-800934bc16a1)

### DBeaver Version

CE 24.2.1.202409221725

### Operating System

Windows 10

### Database and driver

IBM Informix 4.50.10.1

### Steps to reproduce

1. connect to a Informix database
2. open a SQL editor or SQL console
3. change the database thru SQL like : `database sysusers ; `
![dbeaver db select](https://github.com/user-attachments/assets/79867783-1469-410f-a9d8-7094028b32da)

**Comment by E1izabeth** [comment, marked as answer]
Hi, @ceinmart 
Please turn on the setting `Refresh active schema after SQL execution` to make it work
![image](https://github.com/user-attachments/assets/3ebf1bb0-fd0f-474f-86cc-651fd5ffa420)
The default value of this setting changed

**Comment by ceinmart** [comment]
Thanks @E1izabeth , is exactly that...  
Checked and back the old behavior!
